### PR TITLE
Dev/engagement

### PR DESCRIPTION
**Pull request by bartist**

### CODE REVIEW

File: software-development/Improvements.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Improvements
+
+#Engagement
+One of the possible issues in Open Source Software Development is engagement in the project. Commercial development have several mechanisms to ensure the stability of the project, like interface specifications, plans and habits regarding how things are done{{"mockus2002two" | cite}}. Communication can be a substitute for most of these mechanisms, although the "communication-only" is not scalable. The stability of the project engage the participants of the project and ensure a proper end-product. Extrinsic motivation, like money, reputation and peer pressure, can also boost the engagement in the project.
```
Comment (steffansluis) [inline software-development/Improvements.md]: Commercial development **has**

Comment (steffansluis) [inline software-development/Improvements.md]: `although the "communication-only"` is there a word missing here? the "communication-only"...stategy?

Comment (steffansluis) [inline software-development/Improvements.md]: stability of the project **engages** the participants

Comment (steffansluis) [inline software-development/Improvements.md]: end-product ==> end product

File: software-development/Improvements.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Improvements
+
+#Engagement
+One of the possible issues in Open Source Software Development is engagement in the project. Commercial development have several mechanisms to ensure the stability of the project, like interface specifications, plans and habits regarding how things are done{{"mockus2002two" | cite}}. Communication can be a substitute for most of these mechanisms, although the "communication-only" is not scalable. The stability of the project engage the participants of the project and ensure a proper end-product. Extrinsic motivation, like money, reputation and peer pressure, can also boost the engagement in the project.  
+Open Source Software Development does not always have this stucture to build on. Most contributors in OSSD act based on  intrinsic motivation{{"bitzer2007intrinsic" | cite}}{{"roberts2006understanding" | cite}}, like gaining experience, being part of the community or doing it for fun. This type of motivation does not garuantee engagement, but does improve the mental condition while participating and higher quality performance, due to the lack of outside pressure{{"ryan2000intrinsic" | cite}}{{"10.2307/1449618" | cite}}.
```
Comment (steffansluis) [inline software-development/Improvements.md]: Open Source Software Development **(OSSD)** does

Comment (steffansluis) [inline software-development/Improvements.md]: participating and higher quality performance ==> participating and the quality of the performance

File: software-development/Improvements.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+Still, OSSD has some ways to engage the contributors/developers in the process of software-development. One way, like *Netbeans.org* does, is the distribution of "management" positions that are limited to coordination roles{{"jensen2005collaboration" | cite}}. Their task is to determine and announce what needs to be done and wait for volunteers to accept responsibility. This method gives structure without controlling the development process.
```
Comment (steffansluis) [inline software-development/Improvements.md]: `"management" positions` why is management between quotes? I would either explain why the positions aren't exactly management positions, or just rephrase it to avoid the word management

File: software-development/Improvements.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+Still, OSSD has some ways to engage the contributors/developers in the process of software-development. One way, like *Netbeans.org* does, is the distribution of "management" positions that are limited to coordination roles{{"jensen2005collaboration" | cite}}. Their task is to determine and announce what needs to be done and wait for volunteers to accept responsibility. This method gives structure without controlling the development process.  
+Another way is enabling quality control in a project. Thanks to peer review and quick feedback loops, contributors can get a lot of respons on the cod s/he produced. This constant communication between contributors and community drives the contributors to develop better quality code{{"von2012carrots" | cite}}. Thus, code-checking volunteers and contributors can be actively engaged in the quality of the code by giving/reacting to feedback.
```
Comment (steffansluis) [inline software-development/Improvements.md]: cod s/he produced ==> cod**e they** produced

Comment (steffansluis) [inline software-development/Improvements.md]: a lot of respons**es** on

Comment (steffansluis) [inline software-development/Improvements.md]: better quality code ==> **higher quality code** or **better code**

Comment (steffansluis) [inline software-development/Improvements.md]: `Thus, code-checking volunteers and contributors can be actively engaged in the quality of the code by giving/reacting to feedback.`

This sentence is a bit weird in my opinion, since code-checking isn't really a word and the sentence is split by the duality of giving/reacting to feedback. I would rephrase it to be something like:
`Thus, quality control engages both volunteers and contributors to participate in the community.`, or something like that

**Comment by bartist** [comment]
Feedback is processed,
all is rebased

File: software-development/Improvements.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#Improvements
+
+#Engagement
+One of the possible issues in Open Source Software Development is engagement in the project. Commercial development has several mechanisms to ensure the stability of the project, like interface specifications, plans and habits regarding how things are done{{"mockus2002two" | cite}}. Communication can be a substitute for most of these mechanisms, although the "communication-only" approach is not scalable. The stability of the project engages the participants of the project and ensure a proper end product. Extrinsic motivation, like money, reputation and peer pressure, can also boost the engagement in the project.
```
Comment (steffansluis) [inline software-development/Improvements.md]: stability of the project engages the participants **in** the project and ensure**s** a proper end product

File: software-development/Improvements.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#Improvements
+
+#Engagement
+One of the possible issues in Open Source Software Development is engagement in the project. Commercial development has several mechanisms to ensure the stability of the project, like interface specifications, plans and habits regarding how things are done{{"mockus2002two" | cite}}. Communication can be a substitute for most of these mechanisms, although the "communication-only" approach is not scalable. The stability of the project engages the participants of the project and ensure a proper end product. Extrinsic motivation, like money, reputation and peer pressure, can also boost the engagement in the project.  
+Open Source Software Development (OSSD) does not always have this stucture to build on. Most contributors in OSSD act based on  intrinsic motivation{{"bitzer2007intrinsic" | cite}}{{"roberts2006understanding" | cite}}, like gaining experience, being part of the community or doing it for fun. This type of motivation does not garuantee engagement, but does improve the mental condition while participating and the quality of the performance, due to the lack of outside pressure{{"ryan2000intrinsic" | cite}}{{"10.2307/1449618" | cite}}.
```
Comment (steffansluis) [inline software-development/Improvements.md]: does not **guarantee** engagement
